packages/e2e-test-utils-playwright/src/site-editor/toggle-canvas-mode.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Enters the site editor edit mode.
+ *
+ * @this {import('.').SiteEditor}
+ */
+export async function enterEditMode() {
+	await this.page.click( '.edit-site-layout__edit-button' );
```

Comment:
It could go in the `Editor` class if you don't like the idea of inlining it. Maybe with a method name or docs that make it clear it's for the site editor only.

Reply:
I don't like the idea of inlining because it's a very common utility in the site editor. 

Actually, thinking more, I think personally I like it better as it is. I can see us adding utilities to navigate between sidebar items in the site editor as we expand them. (going to templates, template parts, styles, navigation)